### PR TITLE
Add a wrap that notifies about Python package releases on PyPI

### PR DESCRIPTION
**Pull request by fridex**
## Related Issues and Dependencies

Fixes: https://github.com/thoth-station/adviser/pull/1965

## This introduces a breaking change

- [x] No

### CODE REVIEW

File: thoth/adviser/wraps/pypi_release.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
 # thoth-adviser
-# Copyright(C) 2020 - 2021 Fridolin Pokorny
+# Copyright(C) 2021 Fridolin Pokorny
```
Comment (goern) [inline thoth/adviser/wraps/pypi_release.py:3]: @KPostOffice if you are adding significant amout of code, please feel free to add yourself here ;)

Comment (fridex) [reply thoth/adviser/wraps/pypi_release.py:3]: Thanks for your suggestion, Christoph.

**Comment by fridex** [comment]
Will merge this to continue with work, feel free to raise any comments or suggestions.

/approve

**Pull request merged**